src/Passes/libs/QubitAllocationAnalysis/SPECIFICATION.md

```diff
@@ -1,9 +0,0 @@
-# Qubit Allocation Analysis
```

Comment:
It might be worth moving this file to the new location rather than deleting it - alternatively this is short enough that it might make more sense as a doc comment on the code. 

Regarding the renamed .ll files (I can't comment on those files in GitHub), what is the reason that they are kept but renamed to .deprecated? Are they used for any tests?

Reply:
Hmm... that file was supposed to be deleted.

The `.ll` files are tests for the old QirAllocationAnalysis. They are kept as `.deprecated` to prevent them from running, but they are intended to be reinstated once I write tests for the full system.